### PR TITLE
Modern constraints

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

**Comment by saghul** [comment]
@paweldomas This is ready for review.

File: RTCUtil.js [hunk 1]
```diff
@@ -1,35 +1,178 @@
...
+}
+
+function extractNumber(constraints, prop) {
+    const value = constraints[prop];
+    const type = typeof value;
+
+    if (type === 'number') {
+        return Number.parseInt(value);
+    } else if (type === 'object') {
+        for (const v of [ 'exact', 'ideal', 'min ']) {
```
Comment (paweldomas) [inline RTCUtil.js]: how about `max` ?

**Review by paweldomas — DISMISSED** [review]

File: RTCUtil.js (hunk 1 above)
Comment (saghul) [reply RTCUtil.js]: Yeah, I can add it indeed.

**Comment by saghul** [comment]
@paweldomas Updated, PTAL.

**Review by paweldomas — APPROVED** [review]

**Pull request merged**